molecule/commands.py

```diff
@@ -159,6 +159,8 @@ def converge(self, idempotent=False, create_instances=True, create_inventory=Tru
             kwargs.pop('_err', None)
             kwargs['_env']['ANSIBLE_NOCOLOR'] = 'true'
             kwargs['_env']['ANSIBLE_FORCE_COLOR'] = 'false'
+            kwargs['_env']['ANSIBLE_CALLBACK_PLUGINS'] = os.path.join(
+                sys.prefix, 'share/molecule/ansible/plugins/callback/idempotence')
```

Comment:
But what will a user do if they want to add their own callback plugin? Presumably this value can accept a list of paths like most other path config options?